### PR TITLE
Update Question commands of UG

### PR DESCRIPTION
**Pull request by ketamethane**
Includes removal of `liststu`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -19,11 +19,10 @@ more effectively than traditional GUI apps.
         * Adding help tag to a student: [`helpstu`](#adding-help-tag-helpstu)
         * Deleting a student: [`deletestu`](#deleting-a-student-deletestu)
         * Finding a student: [`findstu`](#finding-a-student-findstu)
-        * Listing all students: [`liststu`](#listing-all-students-liststu)
```
Comment (yuxuanleong) [inline docs/UserGuide.md:22]: If I am not wrong after `findstu`, the list will only show a certain number of students, `liststu` will then list all the students again so it is still valid.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -186,16 +185,6 @@ Example:
...
-### Listing all students: `liststu`
-
-Lists all students in the student list.
-
-Format: `liststu`
-
-Example: 
-
-* `liststu`
-
```
Comment (yuxuanleong) [inline docs/UserGuide.md:198]: Might need to trouble you to add the details back for this part too.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -232,8 +222,9 @@ Format: `unmarkq INDEX`
 
 * Marks the question at the specified INDEX as unimportant.
 * The index refers to the index number shown in the displayed question list.
-* The index must be a positive integer 1, 2. 3, …
-*
+* The index must be a positive integer 1, 2, 3, ...
+* The index must be within the number of questions in the  question list. E.g. There are 4 questions. The possible
+  indexes are 1, 2, 3 and 4.
```
Comment (yuxuanleong) [inline docs/UserGuide.md:227]: This is a clearer description! Thank you for adding this.

**Review by yuxuanleong — COMMENTED** [review]
Some minor changes are needed but the rest is all good!

File: docs/UserGuide.md (hunk 1 above)
Comment (ketamethane) [reply docs/UserGuide.md:22]: Added back in

File: docs/UserGuide.md (hunk 2 above)
Comment (ketamethane) [reply docs/UserGuide.md]: Added back in.

**Pull request merged**